utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -31,15 +31,15 @@ struct WriteBatchIndexEntry {
         key_offset(ko),
         key_size(ksz),
         search_key(nullptr) {}
-  WriteBatchIndexEntry(const Slice* sk, uint32_t c)
-      : offset(0),
+  WriteBatchIndexEntry(const Slice* sk, uint32_t c, bool forward_direction)
+      : offset(forward_direction ? 0 : port::kMaxSizet),
```

Comment:
Can we add inline comments of why using max integer is a good value for upset when it is forward direction?

Reply:
Here we set the offset of kMaxSizet but later in comparator() function we compare it against kFlagMin. I know they are equal but using different names for it could be confusing.